build.js

```diff
@@ -210,8 +243,9 @@ function fullbuild () {
   })
 }
 
+// The server function, where the site is exposed through a static file server.
```

Comment:
The server function, where the site is exposed through a static file server **locally**.